kb/migrating-visibility-data-from-es6.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: migrate-visibility-data-from-es6
+title: Migrate visibility data from ES6
+tags:
+  - kb-article
+date: 2022-29-08T00:00:00Z
+---
+
+We added support for [Elasticsearch v7+ (ES7)](https://www.elastic.co/downloads/past-releases/#elasticsearch) in the v1.7.0 update to the [Temporal Server](/clusters#temporal-server).
+ES7 introduces several breaking changes, including the removal of mapping types.
+These changes make ES6 incompatible with ES7.
+
+Upgrade ES6 to ES7, and then use one of the following methods to update your Temporal Server.
+
+:::note
+
+If you are using AWS Elasticsearch, only upgrade to version 7.7.
+
+:::
+
+## Fresh installation
+
+1. Install the latest version of Temporal Server.
+2. Install ES7 as shown in the [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html).
+3. To set the Elasticsearch version in Temporal Server:
+
+- Set the version in the server's static config file:
+
+  ```
+  persistence:
+    datastores:
+      es-visibility:
+        elasticsearch:
+          version: "v7"
+  ```
+
+- Set `ES_VERSION` to `v7` in the pre-built Docker image.
```

Comment:
`v7` has been default for a long time already. So with fresh installation you don't need (3). And I am not sure if this needs to be mentioned at all here as the doc is called "migrating". I would remove entire "Fresh installation" section.